apps/shouldideploy/should_i_deploy.star

```diff
@@ -0,0 +1,66 @@
+"""
+Applet: Should I Deploy
+Summary: Display shouldideploy.today
+Description: Display shouldideploy.today answer.
+Author: humbertogontijo
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+
+SHOULD_I_DEPLOY_URL = "https://shouldideploy.today/api?tz="
+DEFAULT_TIMEZONE = "UTC"
+
+def main(config):
+    tz = config.get("tz", DEFAULT_TIMEZONE)
```

Comment:
Would be great if you could add some caching here. We don't want to hit the API too frequently, and the `main()` function will be called every time a Tidbyt running this app is about to show it in its rotation.

Reply:
Thank you for the review. I've added a 2min cache to reduce api calls